bin/rucio

```diff
@@ -1995,6 +2009,7 @@ You can filter by key/value, e.g.::
     list_rse_usage_parser.set_defaults(which='list_rse_usage')
     list_rse_usage_parser.add_argument(dest='rse', action='store', help='Rucio Storage Element (RSE) name.').completer = rse_completer
     list_rse_usage_parser.add_argument('--history', dest='history', default=False, action='store', help='List RSE usage history. [Unimplemented]')
+    list_rse_usage_parser.add_argument('--per-account', dest='per_account', action='store_true', default=False, help='List RSE usage per account')
```

Comment:
`--per-account` sounds a bit weird to me. I think `--show-accounts` or something similar might be better.